code/src/index.css

```diff
@@ -1,13 +1,94 @@
+* {
+  /* border: 1px red solid; */
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #181818;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.icon-container {
+  position: absolute;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  opacity: 0;
+  transition: 0.4s;
+  height: 100%;
+}
+
+.icon-container:hover {
+  opacity: 1;
+  background-color: rgb(0 0 0 / 43%);
+}
+
+.icon {
+  width: 50px;
+  margin: 4px;
+  filter: invert(1);
+}
+
+.heart,
+.dots {
+  width: 35px;
+}
+
+.play {
+  width: 45px;
+}
+
+.play:hover {
+  width: 50px;
+  transition: 0.2s ease;
+}
+
+.wrapper {
+  max-width: 1500px;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.grid-parent {
+  display: grid;
+  gap: 20px;
+  flex-direction: column;
+  padding: 20px;
+}
+
+@media (min-width: 667px) and (max-width: 1024px) {
+  .grid-parent {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    padding: 20px;
+  }
+
+  .iconContainer {
```

Comment:
We noticed that the heart-icon and dots-icon move when hovering over the play-icon. Maybe you can use this icon container to control the icon's behavior instead of using width on the play button hover state.